publication-commons/src/main/java/no/unit/nva/publication/model/business/Entity.java

```diff
@@ -29,7 +30,7 @@ public interface Entity {
     
     void setIdentifier(SortableIdentifier identifier);
     
-    Publication toPublication(ResourceService resourceService);
+    Publication toPublication(ResourceService resourceService) throws InvalidAssociatedArtifactsException;
```

Comment:
I do not think it should throw Exception. If the conversion is wrong between a database object and the model then it is an IllegalStateException. Also do we want this throws to grow arbitrarily long?